python/tvm/relay/frontend/pytorch.py

```diff
@@ -596,7 +596,7 @@ def tensor_split(self, inputs, input_types):
             raise AssertionError(msg)
 
         if isinstance(inputs[1], torch.Tensor) and not (
-            list(inputs[1].shape) == [] or list(inputs[1].shape) == 1
+            list(inputs[1].shape) == [] or len(inputs[1].shape) == 1
```

Comment:
Perhaps something like this is more concise?

```suggestion
            len(inputs[1].shape) in [0,1]
```

Or the whole condition in line 598 can be simplified to the below one?

```python
if isinstance(inputs[1], torch.Tensor) and len(inputs[1].shape) not in [0,1]:
```

Reply:
Yup. This looks far better and cleaner